tests/vision/checks/distribution/new_labels_test.py

```diff
@@ -0,0 +1,147 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Image Property Drift check tests"""
+from copy import copy
+
+from hamcrest import (
+    assert_that,
+    has_length,
+    has_entries, close_to, has_items
+)
+
+from deepchecks.vision.checks.distribution import NewLabels
+from deepchecks.vision.utils.test_utils import get_modified_dataloader
+from tests.checks.utils import equal_condition_result
+
+
+def get_modification_func_classification(new_labels):
+    def add_new_test_labels(orig_dataset, idx):
+        if idx % 7 == 0:
+            data, label = orig_dataset[idx]
+            return data, new_labels[idx % 5]
```

Comment:
why the mod here?

Reply:
Just a method to change a small fraction of the labels